modules/postgres.nix

```diff
@@ -118,7 +118,8 @@ in
           exit 2
         fi
 
-        systemctl stop postgresql ${lib.concatStringsSep " " cfgu.stopServices}
+        # don't fail when any unit cannot be stopped
+        systemctl stop postgresql ${lib.concatStringsSep " " cfgu.stopServices} || true
```

Comment:
```suggestion
        systemctl stop ${lib.concatStringsSep " " cfgu.stopServices} || true
        systemctl stop postgresql
```
The update should not be continued if postgresql itself can not be stopped.

Reply:
Good catch!